EBH-4608/concat_excel.py

```
@@ -0,0 +1,173 @@
+# create empty dataframe
+merged_excel = pd.DataFrame()
+
+for file_id in all_excels["file_id"]:
+    print(file_id)
+
+    # Use dx cat as dxpy.open_dxfile method would lead to
+    # hyperlinks being removed for all cells
+    cmd = (
+       f"dx cat {file_id} "
+    )
+    output = subprocess.run(cmd, shell=True,
```

Ruff (0.14.7)

`subprocess` call with `shell=True` identified, security issue

(S602)